src/lib/components/admin/Settings/Models/Manage/ManageMultipleOllama.svelte

```
@@ -0,0 +1,26 @@
+<script>
+	import { getContext, onMount } from 'svelte';
```

ESLint

[error] 'onMount' is defined but never used.

(@typescript-eslint/no-unused-vars)